packages/create-cloudflare/turbo.json

```diff
@@ -3,18 +3,20 @@
 	"extends": ["//"],
 	"tasks": {
 		"build": {
+			"passThroughEnv": ["CI"],
 			"env": [
 				"TEST_PM",
 				"TEST_PM_VERSION",
 				"npm_config_user_agent",
-				"CI",
 				"CREATE_CLOUDFLARE_TELEMETRY_DISABLED",
 				"CREATE_CLOUDFLARE_TELEMETRY_DEBUG",
 				"SPARROW_SOURCE_KEY"
 			],
-			"inputs": ["src/**", "templates*/**", "scripts/**", "*", "vitest*"],
 			"outputs": ["dist/**"]
 		},
+		"test:ci": {
+			"dependsOn": ["build"]
+		},
```

Comment:
This is already covered in the root turbo.json, right?

Reply:
Yeah, I'm just adding it here to be a bit more explicit. Happy to remove if you'd prefer